lib/committee/drivers/open_api_3.rb

```diff
@@ -0,0 +1,381 @@
+module Committee::Drivers
+  class OpenAPI3 < Committee::Drivers::Driver
+    # Whether parameters that were form-encoded will be coerced by default.
+    def default_coerce_form_params
+      true
+    end
+
+    # Whether parameters in a request's path will be considered and coerced by
+    # default.
+    def default_path_params
+      true
+    end
+
+    # Whether parameters in a request's query string will be considered and
+    # coerced by default.
+    def default_query_params
+      true
+    end
+
+    def name
+      :open_api_3
+    end
+
+    # Parses an API schema and builds a set of route definitions for use with
+    # Committee.
+    #
+    # The expected input format is a data hash with keys as strings (as opposed
+    # to symbols) like the kind produced by JSON.parse or YAML.load.
+    def parse(data)
+      data = convert_components_to_definitions(data)
+      REQUIRED_FIELDS.each do |field|
+        if !data[field]
+          raise ArgumentError, "Committee: no #{field} section in spec data."
+        end
+      end
+
+      openapi_version = Gem::Version.create(data['openapi'])
+      unless Gem::Version.create('3.0.0') <= openapi_version && openapi_version < Gem::Version.create('4.0.0')
```

Comment:
I think we don't need to check the upper version.

Because OpenAPI 4 has not been released, so we don't know it's breaking changes or not.
If there is breaking changes in v4 the user can use v3, but if there isn't breaking changes, the user can't use v4.

I think that you may allow it until problems occurred.